### DISCUSSION TITLE
Enhance the code/download button for regular people

### DISCUSSION THREAD

**Discussion by Osiris-Team** [community/community#108026] [Repositories]
### Select Topic Area

Product Feedback

### Body

Rename the green 'Code' button to 'Download' again and add the artifacts of the latest release to the dropdown menu that appears after clicking it.

Normal people seem unable to find the releases section and thus ALWAYS download the actual source code.

**Comment by SabineWren** [comment]
People downloading source code instead of releases is by far the biggest cause for support requests. I want to disable that stupid button entirely, as anyone technical enough to want source code can `git clone` instead.
